toonz/sources/common/tapptools/tenv.cpp

```diff
@@ -72,9 +73,17 @@ class EnvGlobals
 #ifdef WIN32
 		return m_registryRoot + varName;
 #else
+  #ifdef APPLE
 		QString settingsPath = QString::fromStdString(getApplicationName()) + QString("_") +
 							   QString::fromStdString(getApplicationVersion()) + QString(".app") +
 							   QString("/Contents/Resources/SystemVar.ini");
+  #else
+		// TODO: use QStandardPaths::ConfigLocation when we drop Qt4
+		QString settingsPath(QDir::homePath());
+		settingsPath.append("/.config/");
+		settingsPath.append(getApplicationName().c_str());
```

Comment:
@johanfleury: `// TODO: use QStandardPaths::ConfigLocation when we drop Qt4` in two lines above.